service/rpc/state.go

```diff
@@ -195,8 +200,13 @@ func (h *Handler) handleTransfer(w http.ResponseWriter, r *http.Request) {
 	}
 	addr, err := types.AccAddressFromBech32(req.To)
 	if err != nil {
-		writeError(w, http.StatusBadRequest, transferEndpoint, err)
-		return
+		// first check if it is a validator address and can be converted
+		valAddr, valErr := types.ValAddressFromBech32(req.To)
+		if valErr != nil {
+			writeError(w, http.StatusBadRequest, transferEndpoint, err)
```

Comment:
same here

Reply:
created a new error for these, please check